politeiawww/codetracker/codetracker.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package codetracker
+
+import (
+	cms "github.com/decred/politeia/politeiawww/api/cms/v1"
+)
+
+// CodeTracker interface for getting Code Stats from a git based code tracking
+// site (Github/Gitlab etc).
+type CodeTracker interface {
```

Comment:
A couple things:
- Put the comments above the function declarations so you don't have comments wrapping around, making them harder to read
- When you package something like this, it should be returning local types, not importing types from the cms API.  So define `UserInformationResult` in this file and then in www you'll need to convert between the codetracker `UserInformationResult` and the cms API `UserInformationResult`.

A couple style nit suggestions, but these are up to you:
- I would abbreviate `Information` as `Info`.  Wide code is harder to read and there's not any pratical difference between `Info` and `Information`
- When you have generic `string` and `int` function arguments on an interface, its helfpul to name them.  Give the reader more info about the functions.

If I was designing this interface, I'd do something like:

```
type CodeTracker interface {
	// Update the user info
	Update(org, repo string) error

	// Request user codestats info
	UserInfo(org, user string, year, month int) (*UserInfoResult, error)
}
```